test/index.js

```diff
@@ -330,6 +349,70 @@ describe('combinator', function() {
 
   });
 
+  describe('A', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.A, 'function');
+      eq(S.A.length, 2);
+    });
+
+    it('returns the result of applying its first argument to its second', function() {
+      eq(S.A(R.inc, 1), 2);
+      eq(R.map(S.A(R.__, 100), [R.inc, Math.sqrt]), [101, 10]);
+    });
+
+    it('is curried', function() {
+      eq(S.A(R.inc).length, 1);
+      eq(S.A(R.inc)(1), 2);
+    });
+
+  });
+
+  describe('C', function() {
+
+    it('is a ternary function', function() {
+      eq(typeof S.C, 'function');
+      eq(S.C.length, 3);
+    });
+
+    it('applies its third and second argument to its first and returns the result', function() {
+      eq(S.C(R.concat, 'foo', 'bar'), 'barfoo');
+      eq(R.map(S.C(R.concat, '!'), ['BAM', 'POW', 'KA-POW']), ['BAM!', 'POW!', 'KA-POW!']);
+    });
+
+    it('is curried', function() {
+      eq(S.C(R.concat).length, 2);
+      eq(S.C(R.concat)('foo').length, 1);
+      eq(S.C(R.concat)('foo')('bar'), 'barfoo');
+    });
+
+  });
+
+  describe('B', function() {
+
+    runCompositionTests(S.B);
+
+  });
+
+  describe('S', function() {
+
+    it('is a ternary function', function() {
+      eq(typeof S.C, 'function');
+      eq(S.C.length, 3);
+    });
+
+    it('applies its third, and its third applied to its second, to its first argument', function() {
```

Comment:
How about this?

```javascript
it('S.S(f, g, x) is equivalent to f(x)(g(x))', ...);
```

It doesn't read as a sentence beginning with `it`, but I don't let this worry me. :)